lib/fathead/pip/fetch.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+baseUrl = 'https://pip.pypa.io/en/stable/reference/'
+import urllib
```

Comment:
Looks like this should be `urllib.request` -- I had to change it to get things working